### ISSUE TITLE
Incorrect alignment for sorting drop-down menus on Chrome

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#2169]
URL: https://staging.webcompat.com/issues?page=1&per_page=50&state=open&stage=all&sort=created&direction=desc
Browser / Version: Chrome Desktop 64.0.3282.167, Chrome Mobile 64.0.3282.137  
Operating System: Window 10 Pro, Google Pixel (Android 8.0.0) - 1080 x 1920 pixels (~441 ppi pixel density)

**Steps to Reproduce:**
1. Navigate to https://staging.webcompat.com/issues?page=1&per_page=50&state=open&stage=all&sort=created&direction=desc
2. Click the "Show 50" drop-down.
3. Observe menu alignment.

**Expected Behavior:**
Menu is correctly aligned and displayed on the right side of the drop-down button.

**Actual Behavior:**
Menu is misaligned and displayed over the drop-down button.
 
**Note**
1. Also reproducible on Mobile.
2. Also reproducible for "Newest" drop-down.
3. Not reproducible on Firefox Desktop&Mobile.
4. Affected area:
```html
<div class="grid-cell x1 js-SearchIssue-sorting">
	<div class="sort-dropdown js-Dropdown js-Dropdown-pagination is-active">
		<button type="button" class="button js-Dropdown-toggle">
			<span class="js-Dropdown-label">Show 50</span>
			<svg class="icon nav-icon" viewBox="0 0 30 30" role="presentation">
				<use href="#svg-chevron-right">dropdown</use>
			</svg>
		</button>
		<ul class="dropdown js-Dropdown-options" aria-hidden="false">
			<li class="js-Dropdown-item nav-dropdown-item">
				<a href="#" class="js-Dropdown-link nav-link" data-params="per_page=25">
        Show 25
      </a>
<!---not needed-->
      </a>
			</li>
		</ul>
	</div>
	<div class="sort-dropdown js-Dropdown js-Dropdown-sort">
		<button type="button" class="button js-Dropdown-toggle">
			<span class="js-Dropdown-label">Newest</span>
			<svg class="icon nav-icon" viewBox="0 0 30 30" role="presentation">
				<use href="#svg-chevron-right">dropdown</use>
			</svg>
		</button>
		<ul class="dropdown js-Dropdown-options" aria-hidden="false">
			<li class="js-Dropdown-item nav-dropdown-item">
				<a href="#" class="js-Dropdown-link nav-link" data-params="sort=created&amp;direction=desc">
        Newest
      </a>
			</li>
<!---not needed-->
		</ul>
	</div>
</div>
```

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

![screenshot_5](https://user-images.githubusercontent.com/12184325/36418227-ebd725cc-1636-11e8-8283-d30e476c38d1.png)

**Comment by zoepage** [comment]
Interesting. Thanks for the report. :)

**Comment by miketaylr** [comment]
@magsout any ideas?

**Comment by zoepage** [comment]
merged #2241
